spring-cloud-task-app-starters-docs/src/main/asciidoc/overview.adoc

```diff
@@ -153,3 +153,184 @@ public class TimestampTaskApplication {
 
 After doing so, you can simply add the additional configuration for the extra features of your application.
 
+=== Developing custom task applications and generating artifacts using the existing tooling
+
+In this section, we will explain how to develop a custom task application and then generate
+maven and docker artifacts for it using the existing tooling provided by the
+spring cloud task app starter infrastructure. For explanation purposes, we will assume that we are creating a new
+task application for a technology named foobar.
+
+* Create a new top level module named spring-cloud-starter-task-foobar
+
+Please follow the instructions above for designing a proper Spring Cloud Task starter. You may also look into the existing
```

Comment:
There are no instructions above to design a Spring Cloud Task starter - that seems to be what is being described in this paragraph.